hal/src/rtl872x/exflash_hal.cpp

```diff
@@ -144,13 +191,17 @@ int hal_exflash_write(uintptr_t addr, const uint8_t* data_buf, size_t data_size)
 __attribute__((section(".ram.text"), noinline))
 int hal_exflash_read(uintptr_t addr, uint8_t* data_buf, size_t data_size) {
     ExFlashLock lk;
+    EnableXip xiplk;
     addr += SPI_FLASH_BASE;
     memcpy(data_buf, (void*)addr, data_size);
+    DCache_CleanInvalidate((uint32_t)data_buf, data_size);
```

Comment:
Does this worsen the performance of reads? Is it absolutely required here?

Reply:
Not really required, it is a little bit overcaution, and every time we call DCache API, we should better make sure that the buffer is 32-byte aligned I'll remove it.